src/rust/src/lazyframe/general.rs

```diff
@@ -1345,4 +1346,21 @@ impl PlRLazyFrame {
             .map_err(RPolarsErr::from);
         Ok(())
     }
+
+    fn sink_json(
+        &self,
+        path: &str,
+        retries: NumericScalar,
+        maintain_order: bool,
+        storage_options: Option<StringSexp>,
+    ) -> Result<()> {
+        let options = JsonWriterOptions { maintain_order };
+        let _retries = <Wrap<usize>>::try_from(retries)?.0;
+
+        let cloud_options = Some(CloudOptions::default());
+
+        let ldf = self.ldf.clone();
+        let _ = ldf.sink_json(path, options, cloud_options);
+        Ok(())
+    }
```

Comment:
This differs quite a bit from rust-polars for now because they're adding all the infrastructure to write partitioned data using `SinkTarget` for instance: https://github.com/pola-rs/polars/blob/4e4768f8b0385138790b1e048fe7617d66f6a4c4/crates/polars-python/src/lazyframe/general.rs#L911

 For now, I'd rather have this simple implementation that can write to a file. We don't use cloud options in other sink functions anyway.